features/issues/components/issue-list/issue-row.tsx

```diff
@@ -72,7 +66,7 @@ export function IssueRow({ projectLanguage, issue }: IssueRowProps) {
       </IssueCell>
       <Cell>
         <Badge color={levelColors[level]} size={BadgeSize.sm}>
-          {levelWords[level]}
+          {capitalize(level)}
```

Comment:
Why did you remove the `levelWords` again? Seemed like the right thing to do

Reply:
I thought it was not as It's the same with the figma design, so I changed it back to original. will fix it haha.